OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -243,31 +233,13 @@ public static bool Exists(string name)
 				return MountedFolders.Any(f => f.Exists(name));
 		}
 
-		static Dictionary<string, Assembly> assemblyCache = new Dictionary<string, Assembly>();
-
 		public static Assembly ResolveAssembly(object sender, ResolveEventArgs e)
 		{
 			foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
 				if (assembly.FullName == e.Name)
 					return assembly;
 
-			var frags = e.Name.Split(',');
```

Comment:
Okay, so this gets called when you have a mod dll (which are loaded manually elsewhere) that links against an additional dll that isn't referenced by the mod.yaml.  This isn't a problem for our mod dlls, but it may be if they want to link against an external library... might need a test case for this.